projects/hmcts-annotation-ui-lib/src/lib/viewers/annotation-pdf-viewer/annotation-pdf-viewer.component.ts

```diff
@@ -0,0 +1,166 @@
+import {Component, OnInit, ViewChild, ElementRef, Input, ChangeDetectorRef, Renderer2, OnDestroy, AfterViewInit, ComponentRef} from '@angular/core';
+import {PdfService} from '../../data/pdf.service';
+import { Subscription } from 'rxjs';
+import {AnnotationStoreService} from '../../data/annotation-store.service';
+import {IAnnotationSet, Annotation} from '../../data/annotation-set.model';
```

Comment:
might be unnecessary but do you want me to change the imports to space or without space but keep all consistent

Reply:
i think space looks neater